src/dns_conf.c

```diff
@@ -97,7 +97,8 @@ struct dns_edns_client_subnet dns_conf_ipv6_ecs;
 
 char dns_conf_sni_proxy_ip[DNS_MAX_IPLEN];
 
-static int _get_domain(char *value, char *domain, int max_dmain_size, char **ptr_after_domain)
+static int _get_domain(char *value, char *domain, int max_dmain_size,
```

Comment:
do not change codestyle, the project use 120 column width

Reply:
Get it. So the `.clang-format` should add `ColumnLimit: 120` because its default limit is `80`.